### ISSUE TITLE
ForceManyBody pushes nodes to 2d layout

### ISSUE THREAD

**Issue by iamtmrobinson** [vasturiano/3d-force-graph#137]
I'm adding forces to the 3d force graph and wanted to have my nodes repel each other more. However if I apply a ForceManyBody with strength -20 the nodes are pushed outwards on a single plane rather than maintaining their location in a 3d space and being pushed further away from each other. Is this expected?

Here is a minimal test case to reproduce what I'm talking about, and you can find a forkable example at [https://codesandbox.io/s/8p3r3j00kj](https://codesandbox.io/s/8p3r3j00kj):

```
import ForceGraph3D from "3d-force-graph";
import * as d3 from "d3";

import data from "./miserables.json";

var myGraph = ForceGraph3D();
myGraph(document.getElementById("app")).graphData(data);

myGraph.d3Force("charge", d3.forceManyBody().strength(-20));
```

**Comment by vasturiano** [comment]
@iamtmrobinson yes, for any additional forces, you should use `d3-force-3d`, if you're in `3D` mode that is.
But, for `forceManyBody`, you don't actually need to add a new force as there's one already in the system, accessible under the name `charge`.

So, you can just do:
```
myGraph.d3Force('charge').strength(-20));
```

**Comment by iamtmrobinson** [comment]
Ah great, thanks for the info @vasturiano
